src/functions.php

```diff
@@ -57,15 +58,16 @@ function server($name, $host = null, $port = 22)
 
 /**
  * @param string $name
+ * @param int $timeout
  * @return Builder
  */
-function localServer($name)
+function localServer($name, $timeout = 60)
```

Comment:
I do not think `localServer` need this option. We do not connecting to localhost.